.github/workflows/system-tests.yml

```diff
@@ -11,7 +11,14 @@ on:
     - cron: '00 04 * * 2-6'
 
 jobs:
+  build-wheels:
```

Comment:
instead can we have this workflow depend on our existing build workflow? that way we can test with the actual artifacts that get published to customers?

yes, ok, re-running probably means we'll build a similar artifact, but testing against the actual real artifact is definitely better,

Reply:
GitLab is the target, but right now their runners aren’t stable enough. Hopefully in the future. For now, trust me, we don’t want them in GL 😓 (and that’s a bigger project than this PR anyway).

For this PR specifically, the more I think about it, the more I’m convinced @nccatoni’s solution is best: we were already recompiling the tracers once per weblog; we’ll still recompile, but only once, and use the action in dd-trace-py. It’s a small step in the right direction.

I fully agree we should compile once and test it, but that’s out of scope for this PR, and it’ll be a slightly bigger project, as we’ve seen 😄.